apps/spruce/src/pages/waterfall/WaterfallErrorBoundary/index.tsx

```diff
@@ -114,7 +114,7 @@ const Container = styled.div`
   box-sizing: border-box;
   position: absolute;
   background-color: rgba(255, 255, 255, 0.8);
-  width: 100%;
+  width: fill-available;
```

Comment:
Nice rule 😁 it doesn't work as well in safari lol but what can you do

Reply:
:shakes_fist_at_safari: [caniuse](https://caniuse.com/?search=fill-available) says it was supported smh :( Replaced it with vw